test/cctest/node_test_fixture.h

```diff
@@ -85,7 +86,9 @@ class NodeTestFixture : public ::testing::Test {
   }
 
   virtual void SetUp() {
-    isolate_ = v8::Isolate::New(params);
+    allocator.reset(node::CreateArrayBufferAllocator());
+    allocator.get_deleter() = &node::FreeArrayBufferAllocator;
```

Comment:
This looks mildly questionable.  Why not e.g.:
```c++
allocator = ArrayBufferUniquePtr(node::CreateArrayBufferAllocator(), &node::FreeArrayBufferAllocator);
```
(Might need a `std::move()`.)

Reply:
I used `reset` mainly to follow the pattern of the other unique_ptr's in that file but this would be much cleaner. I'll update shortly. Thanks